nucypher-core/src/dkg.rs

```diff
@@ -742,7 +733,7 @@ mod tests {
             &ciphertext,
             Some(&Conditions::new("abcd")),
             Some(&Context::new("efgh")),
-            FerveoVariant::SIMPLE,
+            FerveoVariant::Simple,
```

Comment:
Is there an equivalent test case for a request that uses `Precomputed`? You could do one within this same test.